assets/js/feature-tours/shared-key-metrics.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Site Kit by Google, Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/*
+ * Internal dependencies
+ */
+import {
+	VIEW_CONTEXT_MAIN_DASHBOARD,
+	VIEW_CONTEXT_MAIN_DASHBOARD_VIEW_ONLY,
+	VIEW_CONTEXT_ENTITY_DASHBOARD,
+	VIEW_CONTEXT_ENTITY_DASHBOARD_VIEW_ONLY,
+} from '../googlesitekit/constants';
+import { isFeatureEnabled } from '../features';
+
+const sharedKeyMetrics = {
+	slug: 'sharedKeyMetrics',
+	contexts: [
+		VIEW_CONTEXT_MAIN_DASHBOARD,
+		VIEW_CONTEXT_MAIN_DASHBOARD_VIEW_ONLY,
+		VIEW_CONTEXT_ENTITY_DASHBOARD,
+		VIEW_CONTEXT_ENTITY_DASHBOARD_VIEW_ONLY,
+	],
+	gaEventCategory: ( viewContext ) => `${ viewContext }_shared_key-metrics`,
+	checkRequirements: () =>
+		isFeatureEnabled( 'ga4Reporting' ) && isFeatureEnabled( 'userInput' ),
```

Comment:
We're in the process of removing the `ga4Reporting` flag which will also for sure be removed before `userInput` is released so we can safely omit it here.